cfgov/cfgov/settings/base.py

```diff
@@ -764,6 +764,8 @@
     # Controls whether or not to include Qualtrics Web Intercept code for the
     # Q42020 Ask CFPB customer satisfaction survey.
     "ASK_SURVEY_INTERCEPT": [],
+    # Hide archive filter options in the filterable UI
+    "ARCHIVE_FILTER_OPTIONS": [],
```

Comment:
One other minor thing: I think the flag name should be clearer that it disables the archive filter options, so maybe `DISABLE_ARCHIVE_FILTER_OPTIONS`.

Reply:
Done